src/import-service/src/lambda-functions/import-file-parser.ts

```diff
@@ -18,7 +19,19 @@ export const importFileParser = async (event: S3Event) => {
         })
           .createReadStream()
           .pipe(csv())
-          .on('data', console.log)
+          .on('data', (data) => {
+            console.log('data', data)
+            sqs.sendMessage({
+              QueueUrl: process.env.SQS_URL,
+              MessageBody: JSON.stringify(data)
```

Comment:
I think you should send multiple messages for data entries. Now just one message ;)